internal/xds/translator/testdata/out/xds-ir/http-route-zonal-routing-weighted-clusters.endpoints.yaml

```diff
@@ -27,12 +27,18 @@
             address: 3.4.5.6
             portValue: 9090
       loadBalancingWeight: 1
+    loadBalancingWeight: 35
+    locality:
+      region: first-route-dest/backend/1
```

Comment:
If the locality region field doesn't match the envoy instance's I believe this will break zone aware routing. I'd love to learn I'm mistaken here.